confidence_checks/passthrough_adapter.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+from typing import Optional, Any, Sequence
+import pandas as pd
+from core import utils
+from data import recourse_adapter
+
+
+class PassthroughAdapter(recourse_adapter.RecourseAdapter):
+    """A recourse adapter which passes through continuous data without
+    altering it.
+
+    The adapter also optionally simulates rescaling the recourse or adding
+    random noise while interpreting recourse instructions.
+    """
+
+    def __init__(
+        self,
+        label_column: str,
+        perturb_ratio: Optional[float] = None,
+        rescale_ratio: Optional[float] = None,
+        positive_label: Any = 1,
+    ):
+        super().__init__(
+            label_column=label_column, positive_label=positive_label
+        )
+        self.columns = None
+        self.perturb_ratio = perturb_ratio
+        self.rescale_ratio = rescale_ratio
+
+    def fit(self, dataset: pd.DataFrame) -> PassthroughAdapter:
+        super().fit(dataset)
+        self.columns = dataset.columns
+        return self
+
+    def transform(
+        self, dataset: pd.DataFrame
+    ) -> recourse_adapter.EmbeddedDataFrame:
+        df = super().transform(dataset)
+        return df
+
+    def inverse_transform(
+        self, dataset: recourse_adapter.EmbeddedDataFrame
+    ) -> pd.DataFrame:
+        df = super().inverse_transform(dataset)
+        return df
+
+    def directions_to_instructions(
+        self, directions: recourse_adapter.EmbeddedSeries
+    ) -> recourse_adapter.EmbeddedSeries:
+        return directions
+
+    def interpret_instructions(
+        self, poi: pd.Series, instructions: recourse_adapter.EmbeddedSeries
+    ) -> pd.Series:
+        """Interprets the recourse instructions by moving the POI in the
```

Comment:
This description is a bit ambiguous. Might be useful to add a description of `instructions` so it's more explicit how it's structured and how it is used to modify the PoI.